javatests/arcs/core/entity/HandleManagerTestBase.kt

```diff
@@ -552,6 +556,17 @@ open class HandleManagerTestBase {
         assertThat(handle.fetchAll()).containsExactly(modified)
     }
 
+    @Test
+    fun listsWorkEndToEnd() = testRunner {
```

Comment:
this is great. It would be nice to have a similar test using the database (this ones uses ramdisk).
it may be easier to add such test in ReferenceModeStoreDatabaseImplIntegrationTest, just testing the bottom of the stack.

Reply:
This should test all configurations, including the database I think?

Reply:
Looking at the top of the file, all storage keys are ramdisk keys, so I think this only tests ramdisk.

Reply:
ahh, it tests the Storage Service but not against the db.

There's a bunch of tests in DatabaseImplTest that are testing the bottom of the stack - that's probably sufficient? I don't know what adding RMS to that would do given that the RMS is already tested through this test.

Reply:
I would like to insist on this. The interaction between rms and db when using lists is not tested anywhere. I think it would be valuable to have at least a simple test. It shouldn't take long to write one. If you don't want to do it in this patch, it can be done later, but it should be tracked.

Reply:
Done.